### PR TITLE
Feat(gitlab-pipeline): Integrating GitLab Pipeline for Streamlined Project Development

### PR DESCRIPTION
**Pull request by Bankar-Swapnil**
### Functionality:
Pipeline integration for GitLab

### Solution:
required changes done

## WtId:
Dev650

### Risk level:
- [ ] high 
- [ ] medium
- [ ] low

### How to test:
link Gitlab project and go to pipeline page

### CODE REVIEW

**Comment by viv9k** [comment]
These changes should not go to master

File: src/app/body/pipeline/pipeline.component.ts [hunk 1]
```diff
@@ -10,23 +10,29 @@ import { StartServiceService } from "src/app/services/start/start-service.servic
...
 @Component({
   selector: 'app-pipeline',
   templateUrl: './pipeline.component.html',
   styleUrls: ['./pipeline.component.css']
 })
 export class PipelineComponent {
-  componentName = "PIPELINE"
+  componentName = "PIPELINE";
   workflows;
+  workflows2;
```
Comment (viv9k) [inline src/app/body/pipeline/pipeline.component.ts:24]: What is this reference?

**Review by viv9k — COMMENTED** [review]
Code can be optimized

File: src/app/body/pipeline/pipeline.component.ts (hunk 1 above)
Comment (Bankar-Swapnil) [reply src/app/body/pipeline/pipeline.component.ts:24]: yes sir but if we change the team it's not updating the page instantly if we use a single object to show data that's why i  used two different objects to show data

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**